torch/export/_trace.py

```diff
@@ -392,6 +389,37 @@ def _make_module_call_graph(
     return ret
 
 
+def _insert_deferred_runtime_asserts(
+    gm: torch.fx.GraphModule,
+    shape_env: ShapeEnv,
+    name: str,
+):
+    def node_metadata_hook(node: torch.fx.Node) -> None:
+        if node.op != "call_function":
```

Comment:
when this path is triggered?

Reply:
in non-strict export, here: https://github.com/pytorch/pytorch/pull/125414/files/7da850cc69ecd6b13b0c461ead837e0bd5f3604c#diff-4995baf532f07de71ea271ac4a92eaa261b87712be89935ab35797277cb28019R1071-R1075

Reply:
what node.op we will get here in non-strict? looks a bit unreliable because I was expecting the pass to only insert call_function nodes in any case.

Reply:
I guess it should only be call_functions, but because the create_node function is called for all possible nodes, I just ignored the other cases. I could put an assert to make sure for this pass, we're only creating call_function nodes